spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'order'
+require 'pricecalculation'
+require 'send_sms'
+
+describe 'user stories' do
+  let(:subject) { Order.new }
```

Comment:
I don't think lines like this should be used in feature tests, as they are supposed to reflect the inputs from the command line.

Reply:
correct.